spec/controllers/account_controller_spec.rb

```diff
@@ -46,15 +46,15 @@
     end
 
     it 'Returns Non-canceled Illiad Transactions' do
-      get :borrow_direct_redirect
+      get :digitization_requests
```

Comment:
These tests are for the `#illiad_patron_client` method, which is called by both `#borrow_direct_redirect` and `digitization_requests`.